packages/react-day-picker/src/contexts/Modifiers/ModifiersContext.test.tsx

```diff
@@ -0,0 +1,15 @@
+describe('when passing a "selected" custom modifier', () => {
+  test('they should replace the internal "selected" modifiers');
+});
+
+describe('when passing a "disabled" custom modifier', () => {
+  test('they should replace the internal "disabled" modifiers');
+});
+
+describe('when passing a "hidden" custom modifier', () => {
+  test('they should replace the internal "hidden" modifiers');
+});
+
+describe('when passing a "today" custom modifier', () => {
+  test('they should replace the internal "today" modifiers');
```

Comment:
Same as selected…why would we want to let a user specify one think for today and another for today modifier?